tests/e2e/Jenkinsfile

```diff
@@ -0,0 +1,62 @@
+import groovy.json.JsonOutput
+
+/** Send a notice to #fxtest-alerts on irc.mozilla.org with the build result */
+def ircNotification() {
+  def nick = "fxtest${BUILD_NUMBER}"
+  def channel = '#fx-test-alerts'
+  def result = currentBuild.result ?: 'SUCCESS'
+  def message = "Project ${JOB_NAME} build #${BUILD_NUMBER}: ${result}: ${BUILD_URL}"
+  node {
+    sh """
+        (
+        echo NICK ${nick}
+        echo USER ${nick} 8 * : ${nick}
+        sleep 5
+        echo "JOIN ${channel}"
+        echo "NOTICE ${channel} :${message}"
+        echo QUIT
+        ) | openssl s_client -connect irc.mozilla.org:6697
+    """
+  }
+}
+
+pipeline {
+  agent any
+  options { timestamps() }
+  stages {
+    stage('Lint') {
+      steps {
+        sh "tox -c tests/e2e/tox.ini -e flake8"
+      }
+    }
+    stage('Test') {
+      environment {
+        PYTEST_ADDOPTS = -n=${env.PYTEST_PROCESSES ?: 'auto'} --color=yes
+      }
+      steps {
+        timeout(time: 5, unit: 'MINUTES') {
+          wrap([$class: 'AnsiColorBuildWrapper']) {
+            sh "tox -c tests/e2e/tox.ini -e e2e"
+          }
+        }
+      }
+    }
+  }
+  post {
+    always {
+      junit 'tests/e2e/results/*.xml'
+      archiveArtifacts 'tests/e2e/results/*'
```

Comment:
Suggest you change this to: 
```
archiveArtifacts artifacts: 'tests/e2e/results/*', allowEmptyArchive: true
junit 'tests/e2e/results/*.xml'
```

Or even better, If the Lint phase doesn't produce JUnit output, you might move the `junit` step into a `post` block inside the "Test" stage.

See [JENKINS-41239](https://issues.jenkins-ci.org/browse/JENKINS-41239)